templates/user/my_donation.html.twig

```diff
@@ -1,12 +1,28 @@
 {% extends 'adherent/layout.twig' %}
 
+{% block flash_message %}
+    {% for label, messages in app.flashes %}
+        {% for message in messages %}
+            <div class="flash" id="{{ label }}-flashes">
+                <div class="flash__inner">{{ message|trans|raw }}</div>
+            </div>
+        {% endfor %}
+    {% endfor %}
+{% endblock %}
+
 {% block adherent_profile_form %}
     <h2>Mes dons</h2>
 
     {% if donations %}
         <div class="b__nudge--top b__nudge--bottom">
             Merci pour votre engagement à nos côtés ! <br/>
-            Votre dernier don a été fait {{ donations|first.donatedAt|ago }}. <a href="{{ path('donation_index') }}">Faire un nouveau don</a>.
+            Votre dernier don a été fait {{ donations|first.donatedAt|ago }}.
+            {% if subscribed_donations %}
+                <a href="{{ path('donation_subscription_cancel') }}">Arrêter mon don mensuel</a>.
+                Prochaine échéance du don mensuel de {{ subscribed_donations|first.amountInEuros }}€ le {{ subscribed_donations|first.nextDonationAt|date('d M Y') }}
```

Comment:
You display just one donation here, the first one of collection, you don't need to select all donations for displaying just one